saleor/dashboard/graphql/mutations.py

```diff
@@ -31,14 +32,63 @@ def convert_form_errors(form):
 
 class BaseMutation(graphene.Mutation):
     errors = graphene.List(Error)
+    return_field_name = None
```

Comment:
I think this should belong to `MutationOptions` classes. `BaseMutation` is rather for declaring mutation fields. It won't be a problem if `return_field_name` would be duplicated in `ModelFormMutationOptions` and `ModelDeleteMutationOptions`.